frontend/awx/AwxRouter.tsx

```diff
@@ -59,8 +61,9 @@ export function AwxRouter() {
 
         <Route path={RouteObjWithoutPrefix.Jobs} element={<Jobs />} />
         <Route path={RouteObjWithoutPrefix.JobPage} element={<JobPage />} />
+        <Route path={RouteObjWithoutPrefix.CreateSchedule} element={<CreateSchedule />} />
```

Comment:
Instead of a `"coming soon"` UI could you have this route to the `PageNotImplemented` component. Similar to https://github.com/ansible/ansible-ui/blob/main/frontend/awx/AwxRouter.tsx#L74

Reply:
It would be good to add an `Edit Schedule` route as well (also pointing to `PageNotImplemented`) for the list row action button to route to.